cmd/dfdaemon/main.go

```diff
@@ -15,40 +15,97 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
 	"runtime"
 
-	"github.com/sirupsen/logrus"
-
 	"github.com/dragonflyoss/Dragonfly/cmd/dfdaemon/options"
 	"github.com/dragonflyoss/Dragonfly/dfdaemon/initializer"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
```

Comment:
`help` command should be implemented.

Reply:
As for `dfdaemon`, the default implementation is enough. 😄